Workbooks/WorkbooksMetadata.json

```diff
@@ -688,6 +688,45 @@
     "subtitle": "",
     "provider": "Symantec"
   },
+  {
+    "workbookKey": "PulseConnectSecureWorkbook",
+    "logoFileName": "",
+    "description": "Gain insight into Pulse Secure VPN by analyzing, collecting and correlating vulnerability data.\nThis workbook provides visibility into user VPN activities",
+    "dataTypesDependencies": ["PulseConnectSecure"],
+    "dataConnectorsDependencies": [ "PulseConnectSecure" ],
+    "previewImagesFileNames": [ "PulseConnectSecureWhite.png", "PulseConnectSecureBlack.png" ],
+    "version": "1.0",
+    "title": "Pulse Connect Secure",
+    "templateRelativePath": "PulseConnectSecure.json",
+    "subtitle": "",
+    "provider": "Pulse Secure"
+  },
+  {
+    "workbookKey": "InfobloxNIOSWorkbook",
+    "logoFileName": "infoblox_logo.svg",
+    "description": "Gain insight into Infoblox NIOS by analyzing, collecting and correlating DHCP and DNS data.\nThis workbook provides visibility into DHCP and DNS traffic",
+    "dataTypesDependencies": ["InfobloxNIOS"],
+    "dataConnectorsDependencies": [ "InfobloxNIOS" ],
+    "previewImagesFileNames": [ "InfobloxNIOSWhite.png", "InfobloxNIOSBlack.png" ],
+    "version": "1.0",
+    "title": "Infoblox NIOS",
+    "templateRelativePath": "InfobloxNIOS.json",
+    "subtitle": "",
+    "provider": "Infoblox"
+  },
+  {
+    "workbookKey": "SymantecVIPWorkbook",
+    "logoFileName": "symantecVIP_logo.svg",
+    "description": "Gain insight into Symantec VIP by analyzing, collecting and correlating strong authentication data.\nThis workbook provides visibility into user authentications",
+    "dataTypesDependencies": ["SymantecVIP"],
```

Comment:
dataTypesDependencies needs to be Syslog

Reply:
corrections have been made and changes pushed.